GRANTS.yaml

```diff
@@ -67,6 +67,17 @@ CINECA-Project:
     url: https://www.cineca-project.eu
 
 
+dekcd:
+    name: de.kcd
+    github: https://github.com/deKCD
+    joined: ?
+    avatar: ?
```

Comment:
similarly, avatar will by default be the same as the github image, so you can leave this line out unless you want to use a different image than the github profile picture